src/xFactor/base/Application.java

```diff
@@ -0,0 +1,245 @@
+package xFactor.base;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Set;
+
+import javax.persistence.TypedQuery;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import xFactor.config.AppConfig;
+import xFactor.infrastructure.dao.JudgeDao;
+import xFactor.infrastructure.dao.JudgeFavouriteDao;
+import xFactor.infrastructure.dao.ParticipantDao;
+import xFactor.infrastructure.dao.StageDao;
+import xFactor.infrastructure.dao.VoteDao;
+import xFactor.infrastructure.model.Judge;
+import xFactor.infrastructure.model.JudgeFavourite;
+import xFactor.infrastructure.model.Participant;
+import xFactor.infrastructure.model.Person.Gender;
+import xFactor.infrastructure.model.Stage;
+import xFactor.infrastructure.model.Vote;
+import xFactor.service.impl.StageServiceImpl;
+
+@Service
+public class Application {
+	private String name;
+	private int numberOfStages;
+	
+	private Random random = new Random();
+
+	private ArrayList<Judge> allJudges;
+	private ArrayList<Judge> judges;
+	private ArrayList<Participant> allParticipants;
+	private ArrayList<Participant> participants;
+	private ArrayList<Stage> allStages = new ArrayList<Stage>();;
+	private ArrayList<String> qualities;
+
+	@Autowired
+	private StageServiceImpl stageService;
+
+	@Autowired
+	AppConfig config;

```

Comment:
missing access modifier. 
You should not autowire the configurations. They are ran on application start and are meant to initialize beans